db/seeds.rb

```diff
@@ -39,6 +39,18 @@ def generate_users
   end
 
   def generate_items
+    locations = [Location.create(name: 'Capitol Hill'),
+                 Location.create(name: 'Five Points'),
+                 Location.create(name: 'Highland'),
+                 Location.create(name: 'Five Points'),
+                 Location.create(name: 'Lodo'),
+                 Location.create(name: 'City Park'),
+                 Location.create(name: 'Baker'),
+                 Location.create(name: 'Speer'),
+                 Location.create(name: 'Cherry Creek'),
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.